### DISCUSSION TITLE
pdfjs viewer color?

### DISCUSSION THREAD

**Discussion by zutamacka** [quasarframework/app-extension-qpdfviewer#114] [General]
Hello! 

I got the plugin working and I love it! :) Thank you so much!

I have a question though; when I view your examples (eg https://qpdfviewer.netlify.app/examples/pdfjs-example4 ), the top toolbar of the pdf viewer and the frame are nice and dark; and mine are ugly and white. How is that controlled and can I change it? Or is that some dark mode set up on quasar level?

**Comment by hawkeye64** [comment]
PDFjs pretty much takes control and loads it's own CSS. I suspect, for some reason, it's not loading the required CSS, although I have no idea why.
